_posts/2024-11-19-release-notes-3.6.0.md

```diff
@@ -0,0 +1,232 @@
+---
+category: announcement
+permalink: /news/3.6.0
```

Comment:
I think We should rename `3.6.0` to `3.6.2` in all occurrences, and add a link to the publish accident?